parts/linux/cloud-init/artifacts/manifest.json

```diff
@@ -47,7 +47,8 @@
             "1.29.2",
             "1.29.4",
             "1.29.5",
-            "1.30.0"
+            "1.30.1",
+            "1.30.2"
```

Comment:
why are we adding two version to the cache ? is neither of them were previously available ?